### ISSUE TITLE
Create icon for lima project

### ISSUE THREAD

**Issue by yangyang198703** [lima-vm/lima#268]
We'd love to see an official icon for lima project, for referencing it in the graphs, etc.

**Comment by AkihiroSuda** [comment]
Help wanted.

Hints:
- "lima" means "five", or "hand" in a lot of Austronesian languages https://en.wiktionary.org/wiki/Reconstruction:Proto-Austronesian/lima
- "līma" means "file" (carpenter's tool) in Latin https://en.wiktionary.org/wiki/lima#Latin
- "lima" means "lime" in several languages such as Spanish, Portuguese, and Polish https://en.wiktionary.org/wiki/lima#Spanish

**Comment by yangyang198703** [comment]
Should we focus on the  **Li**nux on **Ma**c concept?

**Comment by AkihiroSuda** [comment]
> Should we focus on the **Li**nux on **Ma**c concept?

Not sure. At least we should make sure to avoid causing trademark issues.

**Comment by jandubois** [comment]
As somebody who frequently mistypes `lima` as `lime` I'm partial to a Lime icon (I also like the color). Do a Google image search for [lime icon](https://www.google.com/search?q=lime+icon) to get some inspiration.

Just make sure it looks distinct from the LimeWire icon. The software has been dead for over a decade, but we don't want the association with p2p sharing software.

Theoretically I like "File" (because it is a tool), but files make for rather boring images...

"Five" sounds interesting, but not sure what to make with it.

**Comment by toricls** [comment]
🦙 ([Lama](https://en.wikipedia.org/wiki/Lama_(genus))) ‪🤗‬

**Comment by rothgar** [comment]
I threw this together as an option (or inspriation)

I liked the idea of a lime (circle) and thought the blue from the old mac OS 9 "face" logo would be interesting to mix in 

Lima also made me think of fitting a square peg in a round hole (but in this case linux is a round peg)

![lima](https://user-images.githubusercontent.com/371796/134860991-01731897-5a40-403c-8227-b367d7a72420.gif)

I also animated it which could be cool. I'm not sure why the "i" and "a" have clipping issues but you get the idea.

![lima (1)](https://user-images.githubusercontent.com/371796/134861308-e28daa15-01f1-4844-b9c2-957839c5a4d6.gif)

A green circle with blue square is really basic (I'm not a designer) but could be very simple and identifiable

It's also kinda nice that you can create the logo in unicode which also could be embedded in the terminal output :green_circle: :blue_square:

**Comment by yangyang198703** [comment]
> lime

Love the `lime` idea since Mac => 🍎, Lima => 🍋  (do not have a lime icon on github), it's a fruit vs fruit comparision!

**Comment by dirien** [comment]
+1 on the lama idea from @toricls ! Maybe carrying a container!

**Comment by afbjorklund** [comment]
I think "Linux Machines" would be more appropriate than Linux for Mac, since it works on multiple platforms (unlike WSL)

**Comment by afbjorklund** [comment]
Around here, it's a bus project ;-)

https://www.investingothenburg.com/news/all-news/lindholmen-tests-new-all-one-travel-app

**Comment by afbjorklund** [comment]
And "Lime" is a bike service

**Comment by jandubois** [comment]
> +1 on the lama idea from @toricls ! Maybe carrying a container!

As I stated [before](https://github.com/lima-vm/lima/issues/119#issuecomment-884378562), "Lama" doesn't really work for me. Maybe it is a generational thing, but the camel/llama/alpaca images are synonymous with Perl in my my brain, due to them being the cover images of the O'Reilly books "Programming Perl", "Learning Perl", and "Intermediate Perl" (aka as "the camel book", "the llama book", and "the alpaca book").

The camel book was the most successful O'Reilly book of all time, afaik.

> Anyway, would be more interested in seeing Tux than the Apple in the logo (if anything).

I think the logo must not include any other project/product branding at all to avoid trademark issues.

**Comment by bobhenkel** [comment]
Maybe a lime green lima bean?  Here's a regular lima bean https://en.wikipedia.org/wiki/Lima_bean

**Comment by dirien** [comment]
@jandubois, all good. I understand what you mean. Forgott that they used all this animals on their perl books... 😄 

How about the traditional Peruan hats:

![image](https://user-images.githubusercontent.com/38325136/135034432-a2088a99-deb5-46df-9065-11a060d9a1cf.png)
![image](https://user-images.githubusercontent.com/38325136/135034464-bec40d8e-2cac-430b-853e-856b5faf40e8.png)
As a simple icon! So we have the whole Lima (Peru) theme still going on!

Honestly i would found it nice to keep this link, its unique and different...

**Comment by jandubois** [comment]
> And here's just a big green lima bean sourced from https://www.pngegg.com/en/png-exdwp

I really like it, but we would need explicit permission from the artist before we could use it.

**Comment by dirien** [comment]
@jandubois, but then we put a traditional hat on the bean 😄 that would be awsome!

**Comment by bobhenkel** [comment]
@jandubois Unfortunately the best art I can do is about stick figure level and that's iffy.  I was just using others art to convey my idea.

**Comment by afbjorklund** [comment]
> I really like it, but we would need explicit permission from the artist before we could use it.

You could try https://www.maddie-frost.com/connect.html with regards to the tweet mentioned ?

**Comment by jandubois** [comment]
We may get some original artwork crafted by the Rancher Labs graphical artist; hoping that will work out. We have mgmt approval, but not sure when she'll have time to work on this.

**Comment by StevenACoffman** [comment]
I always thought of Lima beans. :) 
![lima](https://user-images.githubusercontent.com/1942608/135763311-0cf6e4bd-35eb-46f7-91d8-977044e2a6b8.png)

**Comment by jandubois** [comment]
@StevenACoffman Cute, but looks too similar to the Gopher mascot of Golang, imho.

**Comment by StevenACoffman** [comment]
That's exactly where I stole it from. :) I think an actual artist who did something like @dirien  Lima (Peru) hat on a @bobhenkel Lima bean would be much better than my crude amalgamation.

**Comment by AkihiroSuda** [comment]
I noticed now CNCF hosts a project called "Keylime": https://www.cncf.io/projects/keylime/
![keylime](https://keylime.dev/assets/images/keylime.png)


We have to make sure our logo doesn't look like Keylime...

**Comment by AkihiroSuda** [comment]
> Maybe a lime green lima bean? 

Green lima bean is fine, but it should be SVG, and should be an original image to avoid confusion (and to avoid potential copyright concern).

Anybody interested in contribution?

**Comment by dirien** [comment]
hi @jorgeamadosoria, 

as you are very active in SimpleIcons project, could you help us with an Icon for the Lima project?

**Comment by jorgeamadosoria** [comment]
> hi @jorgeamadosoria, 
> 
> as you are very active in SimpleIcons project, could you help us with an Icon for the Lima project?
> 
> 

I have zero information on what this is.
But more importantly, I am not a creative person. My contributions to SI are based on copyong and tweakong existong things, not creating new ones. I'm afraid I lack the skillset.

**Comment by dirien** [comment]
> > hi @jorgeamadosoria,
> > as you are very active in SimpleIcons project, could you help us with an Icon for the Lima project?
> 
> I have zero information on what this is. But more importantly, I am not a creative person. My contributions to SI are based on copyong and tweakong existong things, not creating new ones. I'm afraid I lack the skillset.

Thanks @jorgeamadosoria, i was not sure about this! Thanks again!

**Comment by AkihiroSuda** [comment]
We added an initial logo in #721  (Thanks @jandubois)

![logo](https://raw.githubusercontent.com/lima-vm/lima/aacbc0ec4f3669499f8600954359e8fece959012/docs/images/lima-logo-01.svg)


This logo might be too much simple, but we can improve it incrementally

**Comment by afbjorklund** [comment]
Doesn't really work as an icon, though ? Unless zooming in on the dot or something (🟢)

**Comment by jandubois** [comment]
> Doesn't really work as an icon, though ? Unless zooming in on the dot or something (🟢)

Yeah, technically it is more a [wordmark](https://en.wikipedia.org/wiki/Wordmark) than a logo, but we could shrink it enough to fit into the Github org logo space, which is the immediate use case.

It does not work when you need a really small icon because the shape isn't "square" enough. 😄 

BTW, the file comes from the Rancher Labs graphics person; I only uploaded it (with permission, of course).

**Comment by afbjorklund** [comment]
It does look a bit odd as an icon, compared to the old one.

![lima-gui-logo-icon](https://user-images.githubusercontent.com/10364051/157936505-973e2847-0d33-4fe4-9651-49779a927479.png)

**Comment by jandubois** [comment]
I think the wordmark is supposed to have a white background and should be centered instead of sitting on the baseline.

Now the bikeshedding can begin what the shape of the background should be if not enforced by the application (e.g. in Github it will always be a square).

**Comment by afbjorklund** [comment]
Found this https://www.change.org/p/the-unicode-consortium-we-need-a-lime-emoji

(Regarding the recent change of the containerd and nerdctl "wordmarks" to emojis...)

**Comment by jorgeamadosoria** [comment]
> I think the wordmark is supposed to have a white background and should be centered instead of sitting on the baseline.
> 
> Now the bikeshedding can begin what the shape of the background should be if not enforced by the application (e.g. in Github it will always be a square).

so, this wouldn't be the first logo that is just a wordmark. We have a few of those.
An easy way to create a logo from a wordmark is to just style it with a distinctive font. 

IMO, this is fine. 

As for the shape of the background, you can simply ignore that and just produce a guideline to the effect of "if the icon is going to be used in light color, the background should dark, and if it is dark, the background should bel light" and call it a day. I think that's good enough.

**Comment by afbjorklund** [comment]
I see the logo, I just don't see the icon... But maybe will do something like GitHub does, as suggested.

![lima-github](https://user-images.githubusercontent.com/10364051/158051350-dafdf2ff-a260-463e-b3e9-dedd8480f39f.png)

_(that ", on macOS" seems a bit redundant, especially when you are reading it on Linux)_

**Comment by iboonox** [comment]
What about this ? 
inspired by https://github.com/lima-vm/lima/issues/268#issuecomment-928349370

<img width="1553" alt="CleanShot 2022-03-13 at 19 11 04@2x" src="https://user-images.githubusercontent.com/1584813/158073190-8c0ffe2f-c2de-4eb9-b8ae-33b820fbe621.png">

<img width="1627" alt="CleanShot 2022-03-13 at 19 26 41@2x" src="https://user-images.githubusercontent.com/1584813/158073801-0df90ab9-2b60-4ecc-b96a-4bcce42f966b.png">


<img width="2096" alt="CleanShot 2022-03-13 at 19 12 57@2x" src="https://user-images.githubusercontent.com/1584813/158073252-ea699089-510f-48d5-8345-190e81e3d432.png">

**Comment by AkihiroSuda** [comment]
Thanks @iboonox , but the cursive font accidentally reads "eima" to me.
Can we reuse the current fontwork?
https://raw.githubusercontent.com/lima-vm/lima/v0.9.1/docs/images/lima-logo-01.svg
![logo](https://raw.githubusercontent.com/lima-vm/lima/v0.9.1/docs/images/lima-logo-01.svg)

**Comment by afbjorklund** [comment]
Indeed, this particular bikesheed looks more "green" than "lime" ;-)

**Comment by afbjorklund** [comment]
@iboonox no worries, the green lime looks better anyway...

The lime (green-yellow) and "electric lime" looks cool on a black background, but that's about it.
It is totally illegible on a white (or light) background, something that we found out _after_ changing.

* #485

**Comment by jandubois** [comment]
@iboonox Thank you for all your effort!

From your "variations" I like the last 2 (maybe put little numbers in an image like that, so individual logos are easier to reference).

I prefer the beans with the "swoosh", as that gives them a little bit of 3D effect. Otherwise it just looks like a 2D blob.

But otherwise I prefer the slightly cursive font of the last logo.

I don't like the coloured dot over the `i` when the text is inside a coloured bean (I know I asked for it, but it now seems like a bad idea). The coloured dot looks fine on white background though.

I think for e.g. the Github logo I prefer the text inside the bean, so the logo is easily recognizable by people not yet familiar with lima. For use in documentation and presentation having the plain bean (with "swoosh") and a separate wordmark (with colored dot) looks great too, so maybe we can have both.

It is not clear to me if the font for use inside the bean, and the one for the separate wordmark have to be the same. (I suspect they should be, but I like both the cursive one in the bean, and the minimalistic one used in your last image).

**Comment by afbjorklund** [comment]
Were there any issues with the new artwork ? I think the new icon looks nice, and that the new color is an improvement.

![lima-gui-new-icon](https://user-images.githubusercontent.com/10364051/160240430-472626b5-4a55-41b3-bb06-d251ad144d99.png)

Old color: `#BFFF00`

New color: `#00BF8C`

**Comment by afbjorklund** [comment]
@iboonox created these icons, from your artwork

https://github.com/afbjorklund/lima-gui/blob/0916e2851b3737b9d9bcfe2fd77660ffa4c63ea2/icons/512x512/lima.png

https://github.com/afbjorklund/lima-gui/blob/0916e2851b3737b9d9bcfe2fd77660ffa4c63ea2/icons/scalable/lima.svg

**Comment by afbjorklund** [comment]
I'm unsure about the status of this proposal.

**Comment by jandubois** [comment]
> I'm unsure about the status of this proposal.

I believe the problem is that @iboonox has abandoned the conversation, so we don't know what the copyright status of their images is.

I'm also not sure if we can accept pseudonymous contributions of artwork (we don't accept it for code either).

That's one of the reasons I'm interested in moving the project to CNCF and getting their staff artists to help with this. That establishes clear legal rights to the artwork, plus an organization to defend them, if that ever becomes necessary. This is however a longer road to take; we have not even applied yet, and I believe there is a long backlog of applications that CNCF still has to process...

**Comment by iboonox** [comment]
Sorry I was absent for personal reasons.

Can you please tell me which final version you want to keep so that I can generate the assets on a separate PR ?
Does an Apache 2.0 license suit you ?

**Comment by iboonox** [comment]
> 

@jandubois : To summarize your feedback ?

- keep the last two versions with the 3D effect ( plain bean with "swoosh" )
- Keep the text inside the bean, so the logo is easily recognisable
- Having a plain bean version (with "swoosh") and a separate wordmark for documentation and presentation

I just need to have a validation concerning the font, I see that you prefer the cursive one but I think that the last font respects more the principles of MacOs design (Apple) rather sober and minimalist ?

**Comment by AkihiroSuda** [comment]
"swoosh" may accidentally looks like related to Amazon?

**Comment by jandubois** [comment]
> * keep the last two versions with the 3D effect ( plain bean with "swoosh" )

Well, the last one (with the cursive font) doesn't have the #D effect; I would like to see if it is possible to combine them.

> * Keep the text inside the bean, so the logo is easily recognisable

Yes, I think without the text it is hard to recognize the logo; it could easily look like a blob.

> * Having a plain bean version (with "swoosh") and a separate wordmark for documentation and presentation

Not sure what you mean by "plain version"; I think it should always include the text.
 
> I just need to have a validation concerning the font, I see that you prefer the cursive one but I think that the last font respects more the principles of MacOs design (Apple) rather sober and minimalist ?

I'm confused now, as I thought the last one is the cursive one. I better attach a capture here:

<img width="171" alt="Screen Shot 2022-07-13 at 1 20 34 AM" src="https://user-images.githubusercontent.com/78947/178686252-58b561d1-70d3-4533-ba6c-069cb0550759.png">

This is the image I like best because there are fewer sharp corners in the font, but otherwise still looks relatively plain. The bean is an organic shape, and so I think a font with fluid forms and no right angles looks more natural.

Note that these are just my personal opinions though; not sure how we want to get to consensus.

**Comment by jandubois** [comment]
> "swoosh" may accidentally looks like related to Amazon?

I don't think so, it is just adding some 3D volume to the otherwise flat image. The Amazon image looks quite different.

Note that [Swoosh](https://en.wikipedia.org/wiki/Swoosh) normally refers to the Nike logo, which looks a lot different. But we should stop calling it a "swoosh" because the name is also a Nike brand: https://swoosh.com

**Comment by iboonox** [comment]
> 

By last version I'm referring to : 
<img width="256" alt="image" src="https://user-images.githubusercontent.com/1584813/178689706-07d687c3-f290-41ee-8227-b42ecc69de7a.png">

**Comment by iboonox** [comment]
I'm not sure that GitHub is the best tool for brainstorming ... Would it be possible to have a quick call ? is there a slack lima channel ?

**Comment by AkihiroSuda** [comment]
> quick call

Slack is [here](https://github.com/lima-vm/lima#communication-channels), but I believe it is around 2AM for @jandubois now 😴 
So asynchronous text chat (either on GitHub or on Slack) would be still a good choice.

**Comment by afbjorklund** [comment]
I rendered a larger version of the logo, for use in Podman Desktop:

https://github.com/containers/podman-desktop/pull/3248/files

But gave up on having an icon, so going with the current workarounds.

* ~~https://github.com/afbjorklund/lima-gui/pull/14~~

**Comment by afbjorklund** [comment]
Added an icon for use in Podman Desktop, it's not great but:

<img alt="podman" src="https://github.com/user-attachments/assets/87d3d86d-2aa1-4df3-bbd9-c47dffb60a29" width=128 height=128 />
&nbsp;
<img alt="lima" src="https://github.com/user-attachments/assets/dc4fef53-a5fe-4fe7-840b-17e54118ca66" width=128 height=128 />
&nbsp;
<img alt="docker" src="https://github.com/user-attachments/assets/8ff8d69f-7c92-4ca7-88f0-35b2e57d20ad" width=128 height=128 />

**Comment by afbjorklund** [comment]
I will keep the penguin for GUI/TUI, for now https://github.com/lima-vm/lima/issues/268#issuecomment-950580659

Not too many emojis around https://emojipedia.org/penguin
Was really supposed to go with Tux, as in **Linux** Machines.

Nerdfonts:  Unicode: 🐧 (or just go with a boring "Lima" text)
